src/actions/config.js

```diff
@@ -130,7 +137,9 @@ export function loadConfig() {
 
     try {
       const preloadedConfig = getState().config;
-      const loadedConfig = await getConfig('config.yml', preloadedConfig && preloadedConfig.size > 1);
+      const configUrl = getConfigUrl();
+      console.log(`Netlify CMS using config file: "${configUrl}"`);
```

Comment:
I wonder if we should clarify this a little further. Also, maybe only show it if the link exists since the default behavior is already known. So, this console.log would move into the getConfigUrl function. That way when debugging we know the link was used rather than default. Not detrimental though, just a thought.

`Using config file path: "${configUrl}"`

***Probably don't need to specify Netilify CMS since we already print a version on page load showing Netlify CMS***

Reply:
@talves Awesome! Made the changes!